packages/node/src/subcommands/testing.init.ts

```diff
@@ -13,8 +13,7 @@ export async function testingInit(): Promise<void> {
     const testingService = new TestingService(nodeConfig, project);
     await testingService.run();
   } catch (e) {
-    logger.error(e, 'Testing failed');
-    process.exit(1);
+    exitWithError(`Testing failed,${e}`, logger);
```

Comment:
I cant find where i commented but we should update all the errors to use cause.

`exitWithError(new Error('Testing failed', { cause: e }), logger);`